src/NLog.Raven/RavenTarget.cs

```diff
@@ -69,14 +71,18 @@ public class RavenTarget : TargetWithLayout
         /// </summary>
         [ArrayParameter(typeof(RavenField), "field")]
         public IList<RavenField> Fields { get; set; } = new List<RavenField>();
+        
+        /// <summary>
+        /// Expiration date of document
+        /// </summary>
+        public int ExpirationOffsetInDays { get; set; }
```

Comment:
Maybe rename to `ExpiryOffsetInDays` ? (Easier to spell)

Reply:
Renamed.